### PR TITLE
PMP-2470 | JAN-1302

### PR DESCRIPTION
**Pull request by dtiwarATS**

### CODE REVIEW

File: assets/src/adaptivity/capi.ts [hunk 1]
```diff
@@ -98,12 +99,70 @@ export interface ICapiVariableOptions {
...
   allowedValues?: string[];
   bindTo?: string;
+  shouldConvertNumbers?: boolean;
 }
 
 export interface JanusConditionProperties extends ConditionProperties {
   id: string;
   type?: CapiVariableTypes;
 }
+export const parseArrayWithStringConversion = (val: unknown): unknown[] => {
```
Comment (dtiwarATS) [inline assets/src/adaptivity/capi.ts:109]: Created a new function to handle this just for shake of minimal impact on any other areas.

Comment (bsparks) [inline assets/src/adaptivity/capi.ts:109]: isn't this *without* string conversion? this version is *not* converting the numbers to numbers that are strings

File: assets/src/adaptivity/capi.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { ConditionProperties } from 'json-rules-engine';
-import { isStringArray, parseArray, parseBoolean } from 'utils/common';
+import { isStringArray, parseArray, parseBoolean, parseNumString } from 'utils/common';
```
Comment (bsparks) [inline assets/src/adaptivity/capi.ts:2]: why do we need to add that?

File: assets/src/adaptivity/capi.ts (hunk 1 above)
Comment (bsparks) [inline assets/src/adaptivity/capi.ts:109]: also this one maybe should go into the utils/common and have a unit test to show that it isn't altering the arrays

**Review by bsparks — APPROVED** [review]

**Pull request merged**